src/menu.js

```diff
@@ -19,11 +19,31 @@
       dodecahedron: [1, 2, 3],
       icosahedron: [2, 3, 4],
     },
```

Comment:
I think we can just remove the 'sizes' entries now - it'll automatically be derived from baseSizes on every 'showShapeMenu" call.

Reply:
Good point; will do.